### ISSUE TITLE
fill select values in controller

### ISSUE THREAD

**Issue by grekpg** [APY/APYDataGridBundle#933]
Is possible to fill data to select filter in controller not in annotation ? i have dynamic fields and only in conroller i can push data

**Comment by artscorestudio** [comment]
Yes. You can set column in your controller : https://github.com/artscorestudio/APYDataGridBundle/blob/symfony3/Resources/doc/grid_configuration/add_column.md.

`$myColumn = new BlankColumn(array(
    "selectFrom" => "values",
    "values" => array(
        "dynValue1" => "My dyn value 1",
        "dynValue2" => "My dyn value 2"
    )
))`

**Comment by DonCallisto** [comment]
@grekpg can we close this?
